doc/tutorials/basic/index.md

```diff
@@ -0,0 +1,332 @@
+:::
+
+:::{grid-item-card} {octicon}`clock;2.5em;sd-mr-1` Build a Monitoring Dashboard
+:link: build_monitoring_dashboard
+:link-type: doc
+
+Build a periodically refreshing dashboard to monitor your wind turbines.
+:::
+
+::::
+
+## Organize Content
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`zap;2.5em;sd-mr-1` Use Templates
+:link: templates
+:link-type: doc
+
+Use pre-made templates to easily layout our apps with a header, sidebar, and main area.
+:::
+
+:::{grid-item-card} {octicon}`table;2.5em;sd-mr-1` Layout Content
+:link: layouts
+:link-type: doc
+
+Layout Panel apps as beautifully as efficiently as a wind farm is laid out.
+:::
+
+:::{grid-item-card} {octicon}`screen-full;2.5em;sd-mr-1` Control the Size
+:link: size
+:link-type: doc
+
+We will discover how sizing works in Panel, exploring the differences between inherent sizing, fixed sizing and responsive sizing. Finally we will cover responsive layouts.
+:::
+
+:::{grid-item-card} {octicon}`arrow-switch;2.5em;sd-mr-1` Align Content
+:link: align
+:link-type: doc
+
+We will align our wind turbine images perfectly using `align`, `margin` and `Spacer`s.
+:::
+
+:::{grid-item-card} {octicon}`file;2.5em;sd-mr-1` Build a Report
+:link: build_report
+:link-type: doc
+
+Construct a static Wind Turbine Report, exportable to HTML for efficient distribution to management.
+:::
+
+::::
+
+## Handle User Input
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`person;2.5em;sd-mr-1` Accept Inputs with Widgets
+:link: widgets
+:link-type: doc
+
+Elevate our app's functionality by accepting user input via widgets. Add nice tooltips with explanations to our users or disable widgets for security reasons.
+:::
+
+:::{grid-item-card} {octicon}`arrow-switch;2.5em;sd-mr-1` React to User Input
+:link: pn_bind
+:link-type: doc
+
+React to user input by leveraging `pn.bind`, enabling our users to foster a deeper understanding of wind turbine data through exploration.
+:::
+
+:::{grid-item-card} {octicon}`list-ordered;2.5em;sd-mr-1` Handle State
+:link: state
+:link-type: doc
+
+Be able to build larger and more complex apps by defining and maintaining state via `pn.rx`.
+:::
+
+:::{grid-item-card} {octicon}`checklist;2.5em;sd-mr-1` Build a Todo App
+:link: build_todo
+:link-type: doc
+
+Enable our wind turbine technicians to keep track of their tasks.
+:::
+
+::::
+
+## Show Activity
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`hourglass;2.5em;sd-mr-1` Display Activity with Indicators
+:link: indicators_activity
+:link-type: doc
+
+Show activity with indicators just as rotating blades show the activity of wind turbines.
+:::
+
+:::{grid-item-card} {octicon}`versions;2.5em;sd-mr-1` Update Progressively
+:link: progressive_layouts
+:link-type: doc
+
+Unleash the power of generators and reactive expressions in our Wind Turbine Apps!
+:::
+
+:::{grid-item-card} {octicon}`image;2.5em;sd-mr-1` Build an Image Classifier
```

Comment:
I need some feedback before I can continue with `Build an Image Classifier`.

- Should this example be included in *basic*, *intermediate* or none at all?

Lets assume it should be included

- How can we simplify the code?
- How do we best explain this example? Given solution first and then breaking down code OR building the application in small steps and then giving the solution in the end?